packages/shared/lib/contexts/onboarding/helpers/convertOnboardingProfileToPersistedProfile.ts

```diff
@@ -8,6 +8,8 @@ export function convertOnboardingProfileToPersistedProfile(
 ): IPersistedProfile {
     return {
         ...DEFAULT_PERSISTED_PROFILE_OBJECT,
+        walletPersistedData: {},
```

Comment:
Why clean the information of all the wallets that are already persisted in the profile?

Reply:
What profile? This is code is for when you create an onboardingWallet, there is nothing persisted

Reply:
makes sense :trollface: